palette/src/rgb/rgb.rs

```diff
@@ -953,14 +954,50 @@ where
     }
 }
 
+impl FromStr for Rgb {
```

Comment:
While this may be the more convenient option for many cases, I would still prefer to implement it for `Rgb<Srgb, u8>`. I think that's less surprising than only having it for `f32`. Ideally both would be supported, but then what about if it's a `u16` hex string? Or even larger? So, in the end I think it's better to start small, as you have, but for `Rgb<Srgb, u8>`.

```suggestion
impl FromStr for Rgb<Srgb, u8> {
```

Reply:
Thinking some more about this, I don't think there's any reason to limit it to only `Srgb`.

```suggestion
impl<S: RgbStandard> FromStr for Rgb<S, u8> {
```

It would also be great to have one for `Alpha<Rgb<S, u8>>`.